docs/content/installation/integrations/app-protect-waf-v5/troubleshooting-app-protect-waf.md

```diff
@@ -0,0 +1,136 @@
+---
+docs: DOCS-0000
+doctypes:
+- ''
+title: Troubleshooting NGINX App Protect WAF v5
+toc: true
+weight: 300
+---
+
+This document describes how to troubleshoot problems when using NGINX Ingress Controller and the NGINX App Protect WAF module version 5.
+
+For general troubleshooting of NGINX Ingress Controller, check the general [troubleshooting]({{< relref "troubleshooting/troubleshoot-common" >}}) documentation.
+
+{{< see-also >}} You can find more troubleshooting tips in the NGINX App Protect WAF [troubleshooting guide](/nginx-app-protect/v5/troubleshooting/) {{< /see-also >}}.
+
+## Potential problems
+
+The table below categorizes some potential problems with NGINX Ingress Controller when App Protect WAF module is enabled. It suggests how to troubleshoot those problems, using one or more methods from the next section.
+
+{{% table %}}
+|Problem area | Symptom | Troubleshooting method | Common cause |
+| ---| ---| ---| --- |
+|Start. | The Ingress Controller fails to start. | Check the logs. | Misconfigured policy bundle. |
+|Start | The configuration is not applied. | Check if a policy bundle is compiled using version of the compiler running in NGINX Ingress Controller. | Policy bundle is invalid. |
+|Start | The configuration is not applied. | Check if bundle is present in a volume. | Policy bundle is not present in the mounted volume. |
+|APLogConf, Policy or Ingress Resource. | The configuration is not applied. | Check the events of the APLogConf, Policy and Ingress Resource, check the logs, replace the policy bundle. | Policy bundle is invalid. |
+{{% /table %}}
+
+## Troubleshooting methods
+
+### Check NGINX Ingress Controller and App Protect logs
+
+App Protect logs are part of NGINX Ingress Controller logs when the module is enabled. To check NGINX Ingress Controller logs, follow the steps of [Checking the Ingress Controller Logs]({{< relref "troubleshooting/troubleshoot-common#checking-the-ingress-controller-logs" >}}) of the Troubleshooting guide.
+
+For App Protect specific logs, look for messages starting with `APP_PROTECT`, for example:
+
+```shell
+2020/07/10 11:13:20 [notice] 17#17: APP_PROTECT { "event": "configuration_load_success", "software_version": "2.52.1", "completed_successfully":true,"attack_signatures_package":{"revision_datetime":"2020-06-18T10:11:32Z","version":"2020.06.18"}}
+```
+
+### Check Ingress resource events
+
+Read the topic [Troubleshooting Ingress resources]({{< relref "troubleshooting/troubleshoot-ingress" >}}).
+
+### Check APLogConf events
+
+After you create or update an APLogConf, you can immediately check if the NGINX configuration was successfully applied by NGINX:
+
+```shell
+kubectl describe aplogconf logconf
```

Comment:
I think we need to remove everything which references a appolicy, usersig or logconf custom type